Menu.js

```diff
@@ -57,16 +59,16 @@ const buildMenu = (mainWindow, loadInit) => {
             {
                 label: 'Split Open...',
                 click: (item, focusedWindow) => {
```

Comment:
Seems fair, when I use splits I find it more intuitive to use the Fuzzy File finder or similar to open a new split.